### DISCUSSION TITLE
Ideas for new name of the project

### DISCUSSION THREAD

**Discussion by fujinaga** [DDMAL/VIM#88] [Ideas]
Put your ideas about the new name here. Most of these are from ChatGPT, I list these not because they make good acronyms but to list words that can become part of a good acronym, which should not be a common English word or a common acronym, yet easily pronounceable. 

"Global Musical Instrument Name Guide"
"International Music Instrument Naming Reference" (IMINR)
"Site for International Music Instrument Naming Reference" (SIMINR)
"Worldwide Musical Vocabulary Encyclopedia"
"Cross-Cultural Instrument Lexicon"
"Instrument Names in Multiple Languages" (INiML)
"Polyglot Guide to Musical Instrument Terminology" (PGMIT)
"Global Glossary of Instrument Names" (GGoIN)
"Multinational Musical Instrument Dictionary" (MMID)
"Thesaurus of Musical Instruments" (ToMI)
"Musical Instrument Nomenclature Across Languages" (MINAL)
"Polylingual Musical Instrument Glossary" (PMIG)
"Cultural Harmony: Instruments in Different Tongues"
"Musical Instruments: Multilanguage Encyclopedia" (MIME: too common)
"Worldwide Sound Device Lexicon"
"Global Sonic Tool: Multilingual Instrument Names"
"World Music Instrument Terminology Compendium"
"Cross-Linguistic Encyclopedia of Musical Instruments" (CLEMI)
"Intercontinental Musical Device Dictionary" (IMDD: too close to IMDB)
"Multilingual Compilation of Musical Instrument Names" (MCoMIN)
"Global Encyclopedia of Instrument Names in Different Tongues" 
"Cross-Cultural Compendium of Musical Instrument Terminology"
"Worldwide Repository of Instrument Names in Various Languages" (WRoINVL)
"Instrumental Lexicon Across Different Tongues" (ILADT)
"Intercontinental Catalog of Musical Instrument Names" (ICoMIN)
"Diverse Language Musical Instrument Name Collection" (DLMINC)
"Collection of Musical Instrument Names Across Languages" (COOMINAL)
"Multilingual Dictionary for Names of Musical Instruments" (MDNoMI)
"Editor for Musical Instruments on Wikidata" (EMIW)

**Comment by kyrieb-ekat** [comment]
Galleries, Libraries, Archives, and Museums Musical Instrument Reference
(GLAMmir)("Glamour")

From "GLAM" as a reference term for work conducted around Galleries, Libraries, Archives, and Museums
